### DISCUSSION TITLE
How to get the config from the plugin?

### DISCUSSION THREAD

**Discussion by reod** [yoheimuta/protolint#422] [Q&A] [answered]
I've created a plugin [as described](https://github.com/yoheimuta/protolint/tree/master/_example/plugin) but I'm struggling with getting the config the `protolint` resolved. Any recommendation?

**Comment by yoheimuta** [comment, marked as answer]
You mean you want your plugin to refer to the configuration defined in a file like `.protolint.yaml`?
We don't support this feature. Instead, you can pass some flags to your plugin as described in the example.

**Reply by reod** [reply]
> You mean you want your plugin to refer to the configuration defined in a file like .protolint.yaml?

Exactly. 

It would be awesome to be able to define the config, both for linter and plugins, [in one place](https://github.com/yoheimuta/protolint/blob/1f1793c5dadc57fdfd9430b7bb6ae0f92beb6edb/_example/config/.protolint.yaml#L70) (that's why I asked the question).

> Instead, you can pass some flags to your plugin as described in the example.

Yeah, that's basically what I'm doing now.

Thanks for the reply!
